### ISSUE TITLE
Toolbar Runs Programs From a Different Location Than Everything

### ISSUE THREAD

**Issue by SkyNinja** [stnkl/EverythingToolbar#110]
While trying to use Everything Toolbar, I noticed that some scripts and programs wouldn't run due to launching from a folder different from their own location. At first I thought this might be a limitation with Everything, so I made a simple script to test it.

This is what happens when running the script from Everything:
![2021-02-05](https://user-images.githubusercontent.com/4765557/107100499-28349c00-67da-11eb-847a-28de09c137c2.png)

This is when running from Everything Toolbar:
![2021-02-05 (1)](https://user-images.githubusercontent.com/4765557/107100517-3f738980-67da-11eb-9994-3a238449c6a3.png)

It's also not totally consistent. I haven't figured out what causes it yet, but occasionally, Everything Toolbar will start launching from different locations. I once had it launch the script from my downloads folder.

Also, just so you have all the info, this is the script I used to test this:
```
@echo off
echo Script located in %~dp0
echo Script running from %cd%
pause
```

**Comment by stnkl** [comment]
For me running your file from EverythingToolbar runs it in the same directory where the EverythingToolbar.dll is located. I don't fully understand why it runs in C:\Windows\System32 for you. That should only be the case when it gets run as admin. Did you install EverythingToolbar in your downloads folder?

I changed it to start from the process directory now. It behaves just like Everything for me now. Will be available in the next release. Hopefully that works for you too.

**Comment by SkyNinja** [comment]
> For me running your file from EverythingToolbar runs it in the same directory where the EverythingToolbar.dll is located. I don't fully understand why it runs in C:\Windows\System32 for you. That should only be the case when it gets run as admin. Did you install EverythingToolbar in your downloads folder?

Nope, I have EverythingToolbar at C:\EverythingToolbar. My downloads folder is at D:\Downloads. At the time this happened, I had recently run an installer from my download folder. I can't see how that would influence it, but I can't think of anything else. The installer was for Bandizip, I believe.

I also went ahead and checked if the command prompt is somehow being launched as admin, but it isn't.

**Comment by stnkl** [comment]
Looks like it runs from system32 if you didn't install EverythingToolbar within C:\Users\username. It should be fixed in 0.6.2 stable (which I released just now). Let me know if that is the case for you!

**Comment by SkyNinja** [comment]
Yep, everything is as expected with 0.6.2! No clue what caused the working dir to be downloads that one time, but it only happened once and I could never reproduce it, so it was probably either a weird fluke or I managed to do something strange.

And thanks for all the work you put in on this, it really makes using my PC a whole lot better.
